tests/test_ext.py

```diff
@@ -240,6 +240,22 @@ class T2(Extension):
         assert ext[0].__class__ is T1
         assert ext[1].__class__ is T2
 
+    def test_debug(self):
+        """Test for {% debug %}"""
+        env = Environment(extensions=['jinja2.ext.debug'])
+        tmpl = env.from_string('''Hello{% debug %}Bye''')
+        out = tmpl.render()
+        out = out.replace('&#x27;', "'").replace('&lt;', '<').replace('&gt;', '>')
```

Comment:
Why replace these?

Reply:
If an assert fails, the debug output is much easier to read, and so the tests are easier to maintain.